oas

```diff
@@ -3,273 +3,339 @@ info:
+          name: date
+          schema:
+            type: string
+            format: date
+            example: '2019-05-17'
+            pattern: ^\d{4}-\d{2}-\d{2}$
+          description: The date to use for the retrieval of the indicator. If omitted, the current date will be used.
+          required: false
       responses:
-        200:
+        '200':
           description: Field Monitoring index
           content:
-            image/tiff:
+            image/geotiff:
               schema:
                 type: string
                 format: binary
-                example: image/tiff
-        400:
-          description: Invalid Field URN
+                example: image/geotiff
+        '204':
+          description: Not available Content
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
               example:
-                description: "400 Bad Request: Field URN is invalid"
-                
-        204:
-          description: Not available Content
+                description: '204 No content: Not available Index'
+        '400':
+          description: Invalid Field URN
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
               example:
-                description: "204 No content: Not available Indice"
-        
-        404:
+                description: '400 Bad Request: Field URN is invalid'
+        '404':
           description: Unknown Field URN
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
               example:
-                description: "404 Not Found: Field URN not found"
-        406:
+                description: '404 Not Found: Field URN not found'
+        '406':
           description: Field index out of geospatial bounds
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
               example:
-                description: "406 Not Acceptable: Field index out of geospatial bounds"
-        500:
+                description: '406 Not Acceptable: Field index out of geospatial bounds'
+        '500':
           description: Internal server error
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
               example:
-                description: "500 Internal Server Error" 
-                
-        503:
+                description: 500 Internal Server Error
+        '503':
           description: Service Unavailable
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
               example:
-                description: "503 Service Unavailable"
-  
-
+                description: 503 Service Unavailable
+  /retrieve-ndvi-images:
```

Comment:
This endpoint should be removed. Fused ndvi can be handled as a general ndvi index using the same indices list/search/download endpoints as all other indices. It should not matter that it is a fused index that is computed from satellite imagers (implementation-specific). Unless I am mistaken, for the consumer it is just like an other ndvi; the difference is in the implementation.

Also, the pattern of returning the tiffs in the geojson is not memory friendly. Better to download them one by one as in the "standard" endpoints. In fact, I suggest to slightly modify the result type of search indices:
```
[
  {
    "date": "2019-05-17",
    "indices": [
       {
         "index": "ndvi",
         "download_url": "<ndvi_download_url_for_2019-05-17>"
       }
       ...
    ]
  }
]
```

At this point, you could even remove the oas specs for download url. It never needs to be constructed by hand since it is returned by search.